### PR TITLE
[IMP] l10n_es: Add taxes for 'ISP Bienes de Inversión'

### PR DESCRIPTION
**Pull request by omar7r**
Hi,

When we purchase 'bienes de inversión' with ISP (Inversión de sujetos pasivo) the base and tax amounts must be declared in 303 in two gaps different of current National ISP, that it fits only 'bienes corrientes and servicios corrientes'. In addition, we have to mark the invoice as 'BienInversion' to SII, like @jco-odoo said here https://github.com/OCA/l10n-spain/issues/1793, but it will be the next PR.

Please @pedrobaeza @acysos @rafaelbn @HaraldPanten @AlbertCabedo  etc. check this

Paste AEAT answer about this question:
En cuanto al Modelo 303, una factura recibida correspondiente a un bien de inversión debe con Inversión del Sujeto Pasivo debe declararse tanto en las casillas 12 y 13 del IVA devengado como en las casillas 30 y 31 del IVA deducible (siempre que la adquisición o construcción, en su caso, de dicho bien de inversión se encuentre afecto a la actividad empresarial o profesional y se cumplan el resto de requisitos de deducción de las cuotas de IVA establecidos en los artículos 94 y siguientes de la Ley de IVA).

Regards

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_es/__manifest__.py [hunk 1]
```diff
@@ -11,10 +11,11 @@
 # Albert Cabedo <albert@gafic.com>
 # Olivier Colson <oco@odoo.com>
 # Roberto Lizana <robertolizana@trey.es>
+# Omar Castiñeira <omar@comunitea.com>
 
 {
     "name" : "Spain - Accounting (PGCE 2008)",
-    "version" : "5.0",
+    "version" : "6.0",
```
Comment (pedrobaeza) [inline addons/l10n_es/__manifest__.py:18]: Don't change this file

File: addons/l10n_es/data/account_tax_data.xml [hunk 2]
```diff
@@ -3875,6 +3875,159 @@
             }),
         ]"/>
     </record>
+    <record id="account_tax_template_p_iva4_isp_bi" model="account.tax.template">
+        <field name="description"/> <!-- for resetting the value on existing DBs -->
```
Comment (pedrobaeza) [inline addons/l10n_es/data/account_tax_data.xml:3879]: Not needed

File: addons/l10n_es/data/account_tax_data.xml [hunk 3]
```diff
@@ -3875,6 +3875,159 @@
...
+                'repartition_type': 'tax',
+                'account_id': ref('l10n_es.account_common_477'),
+                'tag_ids': [ref('mod_303_13')],
+            }),
+        ]"/>
+        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'base',
+                'tag_ids': [ref('mod_303_40'), ref('mod_303_14_purchase')],
```
Comment (pedrobaeza) [inline addons/l10n_es/data/account_tax_data.xml:3911]: field 40?

**Comment by pedrobaeza** [comment]
It's a weird case, but the combination may exists, so OK for the general case.

File: addons/l10n_es/__manifest__.py (hunk 1 above)
Comment (omar7r) [reply addons/l10n_es/__manifest__.py]: Ok

File: addons/l10n_es/data/account_tax_data.xml (hunk 2 above)
Comment (omar7r) [reply addons/l10n_es/data/account_tax_data.xml]: I delete it

File: addons/l10n_es/data/account_tax_data.xml (hunk 3 above)
Comment (omar7r) [reply addons/l10n_es/data/account_tax_data.xml:3911]: For refunds I think is correct, the normal taxes (bienes de inversión) use it and the other national ISP too.

File: doc/cla/corporate/comunitea.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+Omar Castiñeira Saavedra omar@comunitea.com https://github.com/omar7r
+
+List of contributors:
+
+Omar Castiñeira Saavedra omar@comunitea.com https://github.com/omar7r
+Javier Colmenero Fernández javier@comunitea.com https://github.com/javierjcf
+Francisco José Sánchez Álvarez kiko@comunitea.com https://github.com/kikosanchez
+Santiago Argüeso Armesto santi@comunitea.com https://github.com/Roodin
+Vicente Gutiérrez Fernández vicente@comunitea.com https://github.com/vicentecom
+Footer
```
Comment (epictete) [inline doc/cla/corporate/comunitea.md:20]:
```suggestion
Vicente Gutiérrez Fernández vicente@comunitea.com https://github.com/vicentecom
```

**Review by epictete — CHANGES_REQUESTED** [review]
Helllo @omar7r 
Thanks for the tax update :slightly_smiling_face: 
Globally, it looks good to me :ok_hand: 
Could you please make a dedicated commit for the CLA ?
Thanks

**Comment by omar7r** [comment]
@epictete Done
Thanks for review

**Review by rafaelbn — APPROVED** [review]
👍

**Review by epictete — APPROVED** [review]
:+1:

**Comment by pedrobaeza** [comment]
Who should be launch the merge? @jco-odoo ?

File: addons/l10n_es/data/account_tax_data.xml [hunk 5]
```diff
@@ -3875,6 +3876,156 @@
...
+            (0,0, {
+                'factor_percent': -100,
+                'repartition_type': 'tax',
+                'account_id': ref('l10n_es.account_common_477'),
+                'tag_ids': [ref('mod_303_15')],
+            }),
+        ]"/>
+    </record>
+    <record id="account_tax_template_p_iva21_isp_bi" model="account.tax.template">
+        <field name="name">IVA 21% Compra con Inversión del Sujeto Pasivo Nacional</field>
```
[the remark this answers is not in the document]
Comment (pedrobaeza) [reply addons/l10n_es/data/account_tax_data.xml:3980]: Being a purchase tax, is not critical that the name is shorter (or to not have a description), as it's not usually printed. But `ISP` can be accepted, and more take it into account that `(bienes de inversión)` is not included and it should be for differentiating it.

[the remark this answers is not in the document]
Comment (jco-odoo) [reply addons/l10n_es/data/account_tax_data.xml:3980]: @pedrobaeza The concern might be the m2m_tags on the invoice line.

[the remark this answers is not in the document]
Comment (pedrobaeza) [reply addons/l10n_es/data/account_tax_data.xml:3980]: OK, let's shorten it then. @omar7r can you?

**Comment by jco-odoo** [comment]
@omar7r Please let us know.  The simplest change might do already.

**Comment by omar7r** [comment]
Done @jco-odoo @pedrobaeza 
Thanks for review

**Review by pedrobaeza — APPROVED** [review]

**Comment by jco-odoo** [comment]
@robodoo r+

**Comment by jco-odoo** [comment]
@robodoo rebase-ff

**Pull request closed without merging**